tests/test_content_client.py

```diff
@@ -12,8 +12,8 @@ def katello_client(request, entities, registration_hostname):
     container_id = subprocess.check_output([
         'podman',
         'run',
-        '--volume',
-        '/dev/log:/dev/log',
+        '--systemd',
+        'always',
```

Comment:
Does the default (autodetection) not work?

Reply:
I cannot seem to get it to reliably work.

Reply:
Hmm ok. I don't mind being explicit and it is something we need anyway. Just curious.